### ISSUE TITLE
Bug?: Hostname is missing in notification email

### ISSUE THREAD

**Issue by certrik** [ail-project/ail-framework#39]
When configuring ...
```
##### Notifications ######
[Notifications]
ail_domain = https://<sub>.<domain>.<tld>:7000 
...
```
to something else than `localhost` the sent out email body doesn't show the ail_domain but nothing e.g. ...
```
item id: submitted/2020/09/04/54cc3ebb-81ce-4609-8cd6-9d9022876022.gz
url: /showsavedpaste/?paste=submitted/2020/09/04/54cc3ebb-81ce-4609-8cd6-9d9022876022.gz
```

**Comment by certrik** [comment]
@Terrtia 
Found the problem. There is a small typo in `RegexTracker.py` at line 61. It reads `ull_item_url = p.config.get("Notifications", "ail_domain") + full_item_url` and should be `full_item_url = p.config.get("Notifications", "ail_domain") + full_item_url` with an `f`. You just lost the `f` somewhere.

**Comment by Terrtia** [comment]
Hi @certrik !

Indeed, Good catch ! 
Do you want to contribute with a pull request ?

**Comment by certrik** [comment]
No, thank you. I am good without a pull request at the moment.

**Comment by Terrtia** [comment]
Thanks for the debugging !

Fixed with 8ef1f05928921a259d16e2fc764c3b95796ea462
